package.json

```diff
@@ -26,8 +26,10 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
+    "big-json": "^2.0.2",
     "bluebird": "^3.5.3",
     "cesium": "^1.54.0",
+    "flush-write-stream": "^1.1.0",
```

Comment:
In general, have we done our due diligence for this and `big-json`?

This looks like a really trivial library, are you sure we want to depend on it?  What specific problem does it solve that you ran into with standard streams?